openmdao/docs/feature_reference/visualization_and_analysis/recording/case_reading.rst

```diff
@@ -0,0 +1,60 @@
+***************************
+Working with Recorded Data
+***************************
+
+A class, `CaseReader`, is provided to read the data from a case recorder file. It will work for any kind of case
+recorder file in OpenMDAO. Currently, OpenMDAO only has a Sqlite case recorder file, but in the future will also have
+an HDF5 case recorder file. `CaseReader` should work for either kind of file as it abstracts away the underlying file
+format.
+
+Here is some simple code showing how to use the `CaseReader` class.
+
+.. code-block:: console
```

Comment:
did we want this test code to be converted to `embed-test` as well? (in another story)